tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -60,13 +60,12 @@ NHLT_BIN=nhlt-sof-lnl-nocodec-fpga-4ch.bin,PASSTHROUGH=true,DMIC_IO_CLK=19200000
 EFX_FIR_PARAMS=passthrough,EFX_IIR_PARAMS=passthrough,EFX_DRC_PARAMS=passthrough"
 
 "sof-hda-generic\;sof-hda-efx-generic-2ch\;\
-HDA_CONFIG=efx,NUM_DMICS=2,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-fir-generic-2ch.bin,\
+HDA_CONFIG=efx,NUM_DMICS=2,\
```

Comment:
ACE platforms can use these with the cAVS blob if they get the NHLT from BIOS, right?

Do we want NHLT in topology for these? To create cAVS and ACE variants with NHLT?

Reply:
I don't think there's need to support non-BIOS NHLT with these topologies. If needed occasionally it's very easy to add ourselves for a build.